### DISCUSSION TITLE
Flux to use ~/.ssh/config file for resolving the fqdn IP mapping Or option to have custom IP mapping

### DISCUSSION THREAD

**Discussion by Baitanik** [fluxcd/flux2#3921] [Q&A]
If we add the fqdn mapping in /etc/hosts, then flux bootstrap all works fine.
but we are trying to automate bootstrap . As multiple people are using same fqdn for installing in different dev clusters, keeping track in /etc/hosts is becoming difficult. 

for git clone/push operation , user specific file ~/.ssh/config we can use, so that different user can have the different IP mapping.. 
NOTE: We can not have different fqdn per clusters for now.. 

cat  ~/.ssh/config
Host ccm.service.xyz.com
    HostName 10.29.10.10
    Port 50023
    IdentityFile ~/.ssh/id_ccm

But flux bootstrap is not honoring this file.
flux bootstrap git --url ssh://git@ccm.service.xyz.com:50023/ccm/flux.git --path clusters/test  --branch main --private-key-file /home/user1/.ssh/id_ccm --registry customrepo/fluxcd

how we can specify custom / user specific fqdn mapping for flux or how to make flux to read this ~/.ssh/config file ? 

flux version
flux: v0.41.1
helm-controller: v0.31.1
kustomize-controller: v0.35.0
notification-controller: v0.33.0
source-controller: v0.36.0

**Comment by tuxillo** [comment]
I've hit the same issue myself, is there any workaround?
